components/script/dom/servoparser/mod.rs

```diff
@@ -585,6 +585,9 @@ impl FetchResponseListener for ParserContext {
                 }
             },
             Some(ContentType(Mime(TopLevel::Text, SubLevel::Xml, _))) => {}, // Handle text/xml
+
+            Some(ContentType(Mime(TopLevel::Application, SubLevel::Xml, _))) => {}, // Handle application/xml
```

Comment:
Can I move the branches for text/xml and application/xml under the last block, i.e. this block? https://github.com/pcpeters/servo/blob/f962fade7e23636f294dba493f6591b45ae86c60/components/script/dom/servoparser/mod.rs#L591

So the way I handle text/xml and application/xml will be similar to the way application/xhtml+xml is currently being handled. 

Would that be ok? Or would it be better to have a separate branch just for text/xml and application/xml like you first suggested?